simulation_server.py

```diff
@@ -398,7 +398,7 @@ def runWebotsInThread(client):
                 if client.webots_process is None:
                     break
                 line = line.rstrip()
-                if line.startswith('webots_1'):  # output from docker-compose's webots service
+                if line.startswith(f'{str(id(self))}-webots-1'):  # output from docker-compose's webots service
```

Comment:
```suggestion
                if line.startswith(f'{str(id(self))}-webots-1'):  # output from docker compose's webots service
```

comment from line 387 should be updated as well